### ISSUE TITLE
Parameterize KeyError when starting from ParamChem parameters

### ISSUE THREAD

**Issue by GerardBCN** [Acellera/htmd#170]
When using --prm and --rtf flags to start from paramchem parameters, the parameterization fails at the charge fitting step with the following output:

 ```
== Minimizing ==

Optimizer: Optimization complete!
Running QM Calculations: 100% (1/1) [#######################################################################################################################] eta --:-- -

 == Charge fitting minimization==

Traceback (most recent call last):
  File "/shared/gerard/anaconda3/bin/parameterize", line 6, in <module>
    sys.exit(htmd.newparameterization.cli.main_parameterize())
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/cli.py", line 135, in main_parameterize
    (score, qm_dipole, mm_dipole) = mol.fitCharges()
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/ffmolecule.py", line 234, in fitCharges
    self._removeCOM()
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/ffmolecule.py", line 208, in _removeCOM
    m =  self._rtf.mass_by_type[ self._rtf.type_by_index[i] ]
KeyError: 'OG2P1'
```

Working directory in http://pub.htmd.org/MES_PSI4.zip

**Comment by GerardBCN** [comment]
The command to run parameterization was: 

`parameterize -m mes.mol2 -n 12 --rtf ff.rtf --prm ff.prm --qmcode PSI4 -c -1`

**Comment by mj-harvey** [comment]
That's working with 1.5.4

**Comment by GerardBCN** [comment]
I got a similar error, so seems that this bug is still not fixed:

```
J. Chem. Theory Comput., 2016, 12 (4), pp 1845-1852.
http://pubs.acs.org/doi/abs/10.1021/acs.jctc.6b00049

You are on the latest HTMD version (unpackaged : /shared/gerard/HTMD/htmd/htmd).

 === List of arguments used ===

      exec:  inline
    theory:  B3LYP
 notorsion:  False
     nomin:  False
   freezeq:  None
      list:  False
    qmcode:  Gaussian
     noesp:  False
forcefield:  all
     basis:  cc-pVDZ
     ncpus:  8
    vacuum:  False
    outdir:  ./
       prm:  ff.prm
       rtf:  ff.rtf
       mol:  cmpd4.mol2
   torsion:  all
    charge:  None

[...]

Traceback (most recent call last):
  File "/shared/gerard/anaconda3/bin/parameterize", line 6, in <module>
    sys.exit(htmd.parameterization.cli.main_parameterize())
  File "/shared/gerard/HTMD/htmd/htmd/parameterization/cli.py", line 191, in main_parameterize
    (score, qm_dipole, mm_dipole) = mol.fitCharges(fixed=fixq)
  File "/shared/gerard/HTMD/htmd/htmd/parameterization/ffmolecule.py", line 240, in fitCharges
    self._removeCOM()
  File "/shared/gerard/HTMD/htmd/htmd/parameterization/ffmolecule.py", line 215, in _removeCOM
    m = self._rtf.mass_by_type[self._rtf.type_by_index[i]]
KeyError: 'CLGR1'
```

I'm using the htmd head version. Current ligand: http://pub.htmd.org/cmpd4.zip

**Comment by j3mdamas** [comment]
I am trying to replicate this, because I don't know at which point this is happening (output cut). I'll let you know tomorrow.

**Comment by j3mdamas** [comment]
@mj-harvey, I don't know how you fixed this before, but what is happening here is that the `.rtf` generated by paramchem does not have the `MASS` entries, and, as such, `_removeCOM` is failing because it relies on them. Did you make some guessing on those, if the original .rtf did not have them?

**Comment by mj-harvey** [comment]
Evidently I fixed a different problem! Does this match the problem I sent you earlier?

**Comment by j3mdamas** [comment]
It's the same problem, as I told Jana. ParamChem .rtf files does not provide MASS entries.

So, we either fail to accept such .rtf files, or use MATCH to guess the MASS entries only?

**Comment by mj-harvey** [comment]
Ok, I guess I can work around that. Masses aren't hard to guess...

**Comment by j3mdamas** [comment]
OK Matt, thanks! Let me know, so I can also update Jana.

**Comment by mj-harvey** [comment]
Fix commited. Please test

**Comment by j3mdamas** [comment]
@GerardBCN @mj-harvey, just tested with the HEAD, and it guesses fine and does not break.

I am going to answer to Jana too and close this issue.

**Comment by j3mdamas** [comment]
It will be available on version 1.5.5

**Comment by j3mdamas** [comment]
3ee96a569cffeda2d3d3aed184b7f2e2ec0ce0b9
